go.mod

```diff
@@ -43,6 +43,7 @@ require (
 	github.com/go-openapi/swag v0.19.14
 	github.com/go-openapi/validate v0.19.8
 	github.com/gobwas/glob v0.2.3
+	github.com/gofrs/uuid v4.2.0+incompatible
```

Comment:
Could we not use `github.com/google/uuid`? It's already an[ indirect dependency](https://github.com/vmware-tanzu/tanzu-framework/blob/9c8711c8e37964ed69d1a55a7e992e61bc061f44/go.mod#L174) and (afaik) provides an identical API.

Reply:
+1. As that's what we're using upstream https://github.com/search?q=org%3Akubernetes+github.com%2Fgoogle%2Fuuid&type=code

Reply:
Thanks Rajath and Naadir. This was a good shout! Created a new PR that does what you suggested: https://github.com/vmware-tanzu/tanzu-framework/pull/1552